architecture/pricing-sources-matrix.md

```diff
@@ -0,0 +1,75 @@
+| [AWS Spot Data Feed](/install-and-configure/install/cloud-integration/aws-cloud-integrations/aws-spot-instances.md) | Kubecost pulls spot prices from a custom feed | More accurate costs in the short-term (48 hours) for Spot nodes | Limited value if billing-integration is configured<br>Requires additional setup with AWS |
+| [Azure Rate Card](../install-and-configure/install/cloud-integration/azure-out-of-cluster/azure-config.md) | Kubecost pulls Azure Rate Card API | More accurate Azure costs in the short-term (48 hours) | Limited value if billing-integration is configured<br>Requires additional setup with Azure |
+| [Alibaba](../install-and-configure/install/provider-installations/alibaba-install.md) | Kubecost pulls Alibaba Rate Card API | Currently the only method to retrieve Alibaba resource pricing |More accurate costs<br>Requires additional setup with Alibaba |
+
+## Diagram
+
+![Cloud Provider Billing Integrated](../images/cloud-bill-diagram.png)
+
+## Details
+
+### Cloud Provider OnDemand API
+
+- [Alibaba](../install-and-configure/install/provider-installations/alibaba-install.md)
+- [AWS EC2](https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/current/us-east-2/index.json)
+- [Azure Rate Card](../install-and-configure/install/cloud-integration/azure-out-of-cluster/azure-config.md)
+
+### Custom Pricing
+
+Set via your *values-kubecost.yaml*, or in the Kubecost UI by going to *Settings*, then under 'Pricing', toggling on 'Enable Custom Pricing', then making adjustments per group:
+
+![UI Custom Pricing Screenshot](/images/custompricing.png)
+
+Prices are monthly. `storage` and miscellaneous network metrics are all per GB.
+
+```yaml
+kubecostProductConfigs:
+  customPricesEnabled: true
+  defaultModelPricing:
+    enabled: true
+    CPU: 28.0 # Single core per month cost
+    spotCPU: 4.86 # Single core per month cost
+    RAM: 3.09 # GB per month cost
+    spotRAM: 0.65 # GB per month cost
+    GPU: 693.50 # per month cost
+    spotGPU: 225.0 # per month cost
+    storage: 0.04 # per GB per month cost
+    zoneNetworkEgress: 0.01 # per GB per month cost
+    regionNetworkEgress: 0.01 # per GB per month cost
+    internetNetworkEgress: 0.12 # per GB per month cost
+```
+
+### Cloud Provider-Billing Integrated
+
+- [Cloud Provider Billing for Reconciliation and Out-of-Cluster Spend](/install-and-configure/install/cloud-integration/README.md)
+  - [AWS Cost and Usage Report](/install-and-configure/install/cloud-integration/aws-cloud-integrations/aws-cloud-integrations.md)
+  - [Azure Cost Export](/install-and-configure/install/cloud-integration/azure-out-of-cluster/azure-out-of-cluster.md)
+  - [Google BigQuery Export](/install-and-configure/install/cloud-integration/gcp-out-of-cluster/README.md)
+
+### CSV Pricing
```

Comment:
Is this section necessary? It feels sparse, plus the detail is already in the other CSV pricing doc. Can we just link the table directly there?

Reply:
killed it.